packages/terra-application-navigation/package.json

```diff
@@ -28,7 +28,7 @@
   "peerDependencies": {
     "react": "^16.8.5",
     "react-dom": "^16.8.5",
-    "react-intl": "^2.4.0",
+    "react-intl": ">=2.4.0, <6.0.0",
```

Comment:
Should we make this `">=2.8.0, <6.0.0"` to be consistent with the other packages?